main.go

```diff
@@ -175,7 +183,7 @@ func Env() map[string]string {
 	return m
 }
 
-func findFailedTests(dirName string, env map[string]string, threshold int) ([]testCase, error) {
+func findFailedTests(dirName string, p params, threshold int) ([]testCase, error) {
```

Comment:
nit: maybe instead of using these loosely coupled functions which in turn pass the params to each specific call make this a struct i.e. `testReporter`, have this be injected with the parameters and each method just uses the struct field?

Reply:
I'll do it in a separated PR to keep changes small

Reply:
https://github.com/stackrox/junit2jira/pull/11